### PR TITLE
create and write markup for Hero component

### PR DESCRIPTION
**Pull request by demirsergen**

### CODE REVIEW

File: src/components/Hero.jsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import skyline from '../images/skyline.svg'
+
+const Hero = () => {
+  return (
+    <section>
+      <div className="hero-content-container">
```
Comment (daleinen7) [inline src/components/Hero.jsx:7]: No need for this div. The above `section` tag will allow us to style this container as needed.

File: src/components/Hero.jsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+        <h4>A CODE FOR AMERICA BRIGADE</h4>
```
Comment (daleinen7) [inline src/components/Hero.jsx:8]: There's a lot of discussion on how to handle eyebrow text. There's no official way to handle it (some argue HTML does not modern designs). But we definitely can't have our `h` tags out of order. So here's how I think we should handle it;

Let's wrap lines 8 and 9 in an `h2` tag (most Hero's contain the page heading and that's what we'll assume is happening here). Then let's wrap "A CODE FOR AMERICA BRIGADE" in a `span` tag with a className of "eyebrow-text". 

And then one last semantic change on this line. Let's keep reusable and simply pass this content as a prop. 

So something like `eyebrowText`.

File: src/components/Hero.jsx [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+        <h1>Making community services work in a digital age</h1>
```
Comment (daleinen7) [inline src/components/Hero.jsx:9]: Let's change this to be dynamic content passed down from props. Something like `mainHeading`.

File: src/components/Hero.jsx [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+        <p>
+          We forge partnerships between neighbors who use technology to improve
```
Comment (daleinen7) [inline src/components/Hero.jsx:11]: Let's pass this paragraph content down via props so it's more reusable.

**Review by daleinen7 — CHANGES_REQUESTED** [review]
Good start. I'd like to set this up to be a little more reusable. Let's display most of the dynamic content through props. 

And then I have a note about the pre heading. I wish I could find the article I read about this but I couldn't find it again. Let me know if you have questions on how or why we're changing it. 

And then I think this will be ready to go.

**Comment by demirsergen** [comment]
There wasn't a comment for the  element but I thought if we are making this component more reusable, maybe we should get the image url via props as well. I might be getting ahead of myself here, so please feel free to correct me if I'm missing something.

**Review by daleinen7 — APPROVED** [review]
Looks good!

**Comment by daleinen7** [comment]
> There wasn't a comment for the `` element but I thought if we are making this component more reusable, maybe we should get the image url via props as well. I might be getting ahead of myself here, so please feel free to correct me if I'm missing something.

From the figma it looks like it will always be the skyline icon, but I'd rather error on the side of making the component reusable so good call on that.

**Pull request merged**